client/src/components/Navbar.js

```diff
@@ -22,10 +21,10 @@ function Navbar() {
             <Link to="/"> <h1 className={styles.name}>bookworms</h1> </Link>
             <div className={styles.links}>
                 <Tooltip title="Add Reviews">
-                    <Link to ="/create"><AddIcon className={styles.button} fontSize="large"/></Link>
+                    <Link to ="/create"><Icon color="primary" className={styles.button} fontSize="large">add</Icon></Link>
                 </Tooltip>
                 <Tooltip title="Sign In">
-                    <SigninIcon className={styles.button} fontSize="large" onClick={() => setSignInShow(true)}/>
+                    <Icon color="primary" className={styles.button} fontSize="large" onClick={() => setSignInShow(true)}>local_library</Icon>
```

Comment:
I can't seem to change the color of icons imported from`material-ui/icons/` so I switched to `material-ui/core/Icon`... if it is not preferable, please let me know and I'll try to work with it again!

Reply:
I think it is OK for now.